VK/VK/ViewControllers/FriendsTableViewController.swift

```diff
@@ -0,0 +1,45 @@
+// FriendsTableViewController.swift
+// Copyright © Roadmap. All rights reserved.
+
+import UIKit
+
+private let reuseIdentifier = "friendCell"
```

Comment:
```suggestion
private let reuseIdentifier = "friendCell"
```
в контролер перенеси